modules/serverless-build-func-kn.adoc

```diff
@@ -8,6 +8,26 @@
 
 Before you can run a function, you must build the function project by using the `kn func build` command. This command creates an OCI container image that can be run locally on your computer or on an {product-title} cluster. The build command uses the function project name and the image registry name to construct a fully qualified image name for your function.
 
+By default, `kn func build` creates a container image using the link:https://buildpacks.io/[CNCF Cloud Native Buildpacks] technology. To use the Red Hat Source-to-Image (S2I) technology instead, supply the `-b` flag:
+
+.Example build command
+[source,terminal]
+----
+$ kn func build -b s2i
```

Comment:
if the sentece is inverted the `-b` wouldn't be required.

Reply:
@jrangelramos You are right. But I inverted it, and showed how to use `-b pack` (seeing how `s2i` is the default).